packages/Webkul/Velocity/src/Resources/lang/hi_IN/app.php

```diff
@@ -0,0 +1,340 @@
+<?php
+
+return [
+    'admin'         => [
+        'system'    => [
+            'velocity' => [
+                'general'  => 'आम',
+                'category'  => 'श्रेणी',
+                'settings'  => 'समायोजन',
+                'extension_name' => 'वेग थीम',
+                'error-module-inactive' => 'चेतावनी: वेग विषय की स्थिति निष्क्रिय है',
```

Comment:
@Sanjaybhattwebkul ,

Translate the velocity to hindi, instead of adding it's meaning.